.travis.yml

```diff
@@ -118,6 +118,6 @@ script:
 
 after_success:
     # Upload to coveralls.
-  - if [ -f build/coverage.info ]; then
-      coveralls-lcov --repo-token ${COVERALLS_TOKEN} build/coverage.info;
+  - if [ -f coverage.info ]; then
```

Comment:
It seems like `after_success` runs in whatever directory `script` ended in, which now is `build`. Coveralls is working again now.